plugins/uefi-capsule/efi/lds/elf_ia32_efi.lds

```diff
@@ -0,0 +1,87 @@
+OUTPUT_FORMAT("elf32-i386", "elf32-i386", "elf32-i386")
+OUTPUT_ARCH(i386)
```

Comment:
Is this linker script really needed ?

Reply:
Agree, we don't build the efi binary for this arch.